pylint_django/transforms/fields.py

```diff
@@ -39,11 +39,7 @@ def apply_type_shim(cls, _context=None):  # noqa
     elif cls.name == 'FloatField':
         base_nodes = scoped_nodes.builtin_lookup('float')
     elif cls.name == 'DecimalField':
-        if sys.version_info >= (3, 5):
-            # I dunno, I'm tired and this works :(
-            base_nodes = MANAGER.ast_from_module_name('_decimal').lookup('Decimal')
-        else:
-            base_nodes = MANAGER.ast_from_module_name('decimal').lookup('Decimal')
+        base_nodes = MANAGER.ast_from_module_name('decimal').lookup('Decimal')
```

Comment:
Don't we have to use the same `try-except ImportError` mechanism like CPython does?